tests/test_trackcompare.py

```diff
@@ -35,41 +35,42 @@ def test_skill(comparer):
 
 def test_skill_vs_spatial_skill(comparer):
     df = comparer.skill()  # to compare to result of .skill()
-    ds = comparer.spatial_skill()
+    ds = comparer.spatial_skill(bins=1)  # force 1 bin only
 
-    assert df.loc["alti"].n == ds.n.values.sum()
-    assert ds.xBin.__len__() == 5
-    assert ds.yBin.__len__() == 5
-    # assert ds.coords._names == {"xBin","yBin"}  # TODO: Why return "observation" a by, when n_obs==1?
+    assert df.loc["alti"].n == ds.n.values
```

Comment:
https://github.com/DHI/fmskill/pull/27#discussion_r612698376 and https://github.com/DHI/fmskill/pull/27#discussion_r612704160